onboarding/asset-management-directions.md

```diff
@@ -0,0 +1,126 @@
+
+To make life easier, we made [some python scripts](https://github.com/NutritionalLungImmunity/data-management) where you can choose tags from a GUI and it will write to a file that you can copy and paste onto the information tab.
+
+### Girder asset tags
+
+
+| Tag       | Include if         |
+| ------------- |:-------------:|
+|NLI:POSTER| Asset is a poster |
+|NLI:DOCUMENTATION| Asset is related to documentation for the project |
+|NLI:SLIDES| Asset is for a presentation in slide format |
+|NLI:EXP.DATA| Asset is part of an experiment related to the NLI project|
+|NLI:METADATA| Asset is metadata associated to a dataset |
+|NLI:EXP.PROTOCOL| Asset is metadata associated to a dataset |
+|NLI:TRANSCRIPTOMICS| Asset is related to a transcriptomic experiment |
+|NLI:TISSUESCALE| Asset is related to data at the tissue scale |
+|NLI:INTRACELLULARSCALE| Asset is related to data at the intracellular scale |
+|NLI:RESULTS|Asset presents results for an analysis or for model|
+|NLI:ABSTRACT|Asset is an abstract for a presentation|
+|NLI:MODEL|Asset is associated with the building of a model|
+|NLI:MDMACROPHAGE|Asset is associated with the cell type|
+|NLI:MDDC|Asset is associated with the cell type|
+|NLI:ALVEOLARMACROPHAGE|Asset is associated with the cell type|
+|NLI:NEUTROPHIL|Asset is associated with the cell type|
+|NLI:ALVEOLAREPITHELIALCELL|Asset is associated with the cell type|
+|NLI:ENDOTHELIALCELL|Asset is associated with the cell type|
+|NLI:LABORATORYNOTEBOOK|Asset is a laboratory notebook|
+|NLI:LASTAUTHOR:LuisSordoVieira|Last/Senior author(s)|
+|NLI:LASTAUTHOR:ReinhardLaubenbacher|Last/Senior author(s)|
+|NLI:LASTAUTHOR:BanditaAdhikari|Last/Senior author(s)|
+|NLI:LASTAUTHOR:JosephMasison|Last/Senior author(s)|
+|NLI:LASTAUTHOR:HenriqueDeAssis|Last/Senior author(s)|
+|NLI:LASTAUTHOR:YuMei|Last/Senior author(s)|
+|NLI:LASTAUTHOR:BornaMehrad|Last/Senior author(s)|
+|NLI:LASTAUTHOR:MichaelGrauer|Last/Senior author(s)|
+|NLI:LASTAUTHOR:WilliamSchroeder|Last/Senior author(s)|
+|NLI:LASTAUTHOR:NingYang|Last/Senior author(s)|
+|NLI:LASTAUTHOR:BrianHelba|Last/Senior author(s)|
+|NLI:LASTAUTHOR:JonathanBeezley|Last/Senior author(s)|
+|NLI:FIRSTAUTHOR:LuisSordoVieira|First author(s)|
+|NLI:FIRSTAUTHOR:ReinhardLaubenbacher|First author(s)|
+|NLI:FIRSTAUTHOR:BanditaAdhikari|First author(s)|
+|NLI:FIRSTAUTHOR:JosephMasison|First author(s)|
+|NLI:FIRSTAUTHOR:HenriqueDeAssis|First author(s)|
+|NLI:FIRSTAUTHOR:YuMei|First author(s)|
+|NLI:FIRSTAUTHOR:BornaMehrad|First author(s)|
+|NLI:FIRSTAUTHOR:MichaelGrauer|First author(s)|
+|NLI:FIRSTAUTHOR:WilliamSchroeder|First author(s)|
+|NLI:FIRSTAUTHOR:NingYang|First author(s)|
+|NLI:FIRSTAUTHOR:BrianHelba|First author(s)|
+|NLI:FIRSTAUTHOR:JonathanBeezley|First author(s)|
+
+
+#### That's not all
+
+If you feel up to the challenge, you can also write your own python scripts to upload/download data from our Girder collection.
```

Comment:
seems a little out of place. what does this relate to? when should they write scripts?

Reply:
maybe this is ok. i was reading out of order, and now i see the section above.